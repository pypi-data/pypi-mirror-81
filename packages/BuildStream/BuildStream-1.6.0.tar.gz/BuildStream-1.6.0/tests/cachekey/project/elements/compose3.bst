kind: compose
depends:
- filename: elements/import1.bst
  type: build

config:
  include-orphans: False
