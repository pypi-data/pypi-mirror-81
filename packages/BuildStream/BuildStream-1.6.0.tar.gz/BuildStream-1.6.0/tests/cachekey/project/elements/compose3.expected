b7c821b8fd006069ae2b55b855be9dde6f3f3ce141dd546a2bce64b0bde9e93a